routes/web.php

```diff
@@ -40,10 +40,11 @@
 Route::group(['middleware' => 'auth'], function () {
     Route::middleware('can:admin')->group(function () {
         Route::get('admin', [AdminController::class, 'index'])->name('admin');
-        Route::get('photos', [AdminController::class, 'photos'])->name('photos');
         Route::post('logout', [SessionsController::class, 'destroy'])->withoutMiddleware(['auth']);
-        Route::get('/admin/illustrations', [AdminController::class, 'illustrations'])->name('illustrations');
-        Route::post('/admin/illustrations/store', [AdminController::class, 'store'])->name('illustrations.store');
-        Route::delete('admin/photos/{photo}', [AdminController::class, 'destroy']);
+        Route::match(['get', 'post'], '/admin/illustrations', [AdminController::class, 'illustrations'])->name('illustrations');
```

Comment:
Tu va revoir le fonctionnement je suppose. Normalement t'as juste besoin de GET (comme vu ensemble)

Reply:
Done! moved back to GET method and deleted @csrf token. Discovered an interesting method to ensure pagination in the search results  https://laravel.com/docs/11.x/pagination#appending-query-string-values that helped to simplify my controller method